src/BottomComponent.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import './BottomComponent.css';
+import { connect } from 'react-redux';
+import { getTrack } from './Utils';
+import { toogleAction } from './actions/trackActions';
+
+class BottomComponent extends React.Component {
```

Comment:
To mogłoby być functional componentem. Zmiana nie jest wymagana, ale jedynie jako informacja, że to jest dobry kandydat ;)

Reply:
zgadza sie bo nie ma zadnych dodatkowych metod poza render()